src/pages/[platform]/build-a-backend/add-aws-services/analytics/personalize-recommendations/index.mdx

```diff
@@ -0,0 +1,149 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Personalized recommendations',
+  description: 'Amazon Personalize can create recommendations by using event data, historical data, or a combination of both. The event data can then be used to create recommendations.',
+  platforms: [
+    'javascript',
+    'angular',
+    'nextjs',
+    'react',
+    'vue',
+    "react-native",
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
+        'javascript',
+        'nextjs',
+        'react',
+        'vue',
+        'angular'
+      ],
+      canonicalPath: '/[platform]/build-a-backend/add-aws-services/analytics/personalize-recommendations/'
+    }
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+Amazon Personalize can create recommendations by using event data, historical data, or a combination of both. The event data can then be used to create recommendations.
+
+To record event data, you need the following:
+
+* [A dataset group](https://docs.aws.amazon.com/personalize/latest/dg/recording-events.html#event-dataset-group)
+* [An event tracker](https://docs.aws.amazon.com/personalize/latest/dg/recording-events.html#event-get-tracker).
```

Comment:
These links go to the same page and for me it does not navigate to the heading. Is this intentional?